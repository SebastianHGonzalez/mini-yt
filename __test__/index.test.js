/* eslint-env jest */

import React from "react";
import { render } from "@testing-library/react";
import { ThemeProvider } from "styled-components";
import "jest-styled-components";

import theme from "constants/theme.js";
import IndexPage from "pages/index.jsx";

describe("With React Testing Library", () => {
  it('Shows "My page"', () => {
    const { getByText } = render(
      <ThemeProvider theme={theme}>
        <IndexPage />
      </ThemeProvider>
    );

    expect(getByText("My page")).not.toBeNull();
  });
});

describe("With React Testing Library Snapshot", () => {
  it("Should match Snapshot", () => {
    const { asFragment } = render(
      <ThemeProvider theme={theme}>
        <IndexPage />
      </ThemeProvider>
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
