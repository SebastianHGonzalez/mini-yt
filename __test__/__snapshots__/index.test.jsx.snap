// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With React Testing Library Snapshot Should match Snapshot 1`] = `
<DocumentFragment>
  .c0 {
  font-size: 50px;
  color: #0070f3;
}

<h1
    class="c0"
  >
    My page
  </h1>
</DocumentFragment>
`;
